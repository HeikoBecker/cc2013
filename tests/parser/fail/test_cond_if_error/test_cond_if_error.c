char blub(char c, int b){
	int a;
	while(a == b){
	}
	{}
	;
	{;};
	{
	if (a; b){
	}
	}
}
