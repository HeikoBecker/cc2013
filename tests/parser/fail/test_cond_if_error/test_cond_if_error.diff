1c1,3
< test_cond_if_error.c:8:7: error: Expected ')' but found ';'
---
> test_cond_if_error.c:9:7: error: Expected ) but got ;
> 	if (a; b){
> 	     ^^^^
