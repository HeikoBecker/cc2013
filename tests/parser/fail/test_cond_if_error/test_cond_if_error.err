test_cond_if_error.c:9:7: error: Expected ) but got ;
	if (a; b){
	     ^^^^
1 error(s)
