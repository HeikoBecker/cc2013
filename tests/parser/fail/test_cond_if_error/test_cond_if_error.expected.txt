test_cond_if_error.c:8:7: error: Expected ')' but found ';'
1 error(s)
